ECart/CartItem.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace ECart
+{
+    class CartItem
+    {
+        Product item;
+        int Quantity;
```

Comment:
Do not use Pascal Case for non public properties. use quantity.

Reply:
Resolved